prime-router/src/main/resources/db/migration/V43__add_fhir_processing_to_task_actions_and_action_log.sql

```diff
@@ -0,0 +1,15 @@
+/*
+ * The Flyway tool applies this migration to create the database.
+ *
+ * Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+ * use VARCHAR(63) for names in organization and schema
+ *
+ * Copy a version of this comment into the next migration
+ *
+ */
+
+/*
+ * Add fhir_processing support for tracking and logging.
+ */
+ALTER TYPE public.task_action ADD VALUE 'fhir_processing';
+ALTER TYPE public.action_log_scope ADD VALUE 'fhir_processing';
```

Comment:
Postgres is kinda awesome about that and allows adding values to enums, `V25` and `V29` also added values to that column. For safety I'm appending it to the end instead of specifying some other location for it.

Throwing out those scope changes, looking at the logic Report is probably the correct scope for this.